theme.json

```diff
@@ -236,6 +236,11 @@
 					"fontFamily": "Cardo",
 					"name": "Cardo",
 					"slug": "heading"
+				},
+				{
+					"fontFamily": "-apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen-Sans, Ubuntu, Cantarell, 'Helvetica Neue', sans-serif",
+					"name": "System Font",
+					"slug": "body-alt"
```

Comment:
we were using the slugs body/heading so patterns using specific fonts wouldn't have a slug with the name of the font, since that doesn't make sense when you change variations. In this case, since this is a font for end users and not for theme developers, we can keep the slug aligned with the name of the font

Reply:
Changed slug to 'system-font'